# A change in OpenSSL changes the hash that is expected.  This updates the hash.
# This will not be sent up stream since it is already fixed in later versions.
diff -ur orig_Net-SSLeay-1.52/t/local/33_x509_create_cert.t Net-SSLeay-1.52/t/local/33_x509_create_cert.t
--- orig_Net-SSLeay-1.52/t/local/33_x509_create_cert.t	2015-02-05 10:23:55.296178177 -0800
+++ Net-SSLeay-1.52/t/local/33_x509_create_cert.t	2015-02-05 10:27:24.727716789 -0800
@@ -104,7 +104,7 @@
   like(my $key_pem4 = Net::SSLeay::PEM_get_string_PrivateKey($pk,"password",$alg2), qr/-----BEGIN (ENCRYPTED|RSA) PRIVATE KEY-----/, "PEM_get_string_PrivateKey+passwd+enc_alg");
   
   is(Net::SSLeay::X509_NAME_print_ex($name), "O=Company Name,C=UK,CN=Common name text X509", "X509_NAME_print_ex");  
-  is(unpack("H*",Net::SSLeay::X509_NAME_digest($name, $sha1_digest)), "044d7ea7fddced7b9b63799600b9989a63b36819", "X509_NAME_digest");
+  is(unpack("H*",Net::SSLeay::X509_NAME_digest($name, $sha1_digest)), "ce83889f1beab8e70aa142e07e94b0ebbd9d59e0", "X509_NAME_digest");
   
   ok(my $ext_idx = Net::SSLeay::X509_get_ext_by_NID($x509, &Net::SSLeay::NID_ext_key_usage), "X509_get_ext_by_NID");
   ok(my $ext = Net::SSLeay::X509_get_ext($x509, $ext_idx), "X509_get_ext");
